doc/nrf/ug_bt_mesh_discoverable_dm_app.rst

```diff
@@ -0,0 +1,35 @@
+.. ug_bt_mesh_discoverable_dm_app:
+
+Discovering Bluetooth mesh device nodes in nRF Connect Device Manager
+#####################################################################
+
+If your Bluetooth mesh device node is not visible in the list of available devices when using the `nRF Connect Device Manager`_ mobile app (for example when performing FOTA upgrades), it is because the device is not advertising the SMP service UUID and the filter that only shows devices advertising this service is enabled.
+The device can still be discovered through a service discovery, for example using the `nRF Connect for Mobile`_ app.
+
+To make sure your device is visible in the `nRF Connect Device Manager`_ mobile app, do one of the following:
+
+* Disable the filter that only shows devices advertising the SMP service UUID in the `nRF Connect Device Manager`_ mobile app.
+* Make the device node advertise the service by enabling and configuring the extended advertising, and thus be discoverable by the `nRF Connect Device Manager`_ mobile app with the filter enabled.
+
+Disabling the filter
+********************
+
+To disable the filter in the `nRF Connect Device Manager`_ mobile app, do the following steps:
+
+1. Tap the :guilabel:`Filter` button at the right top corner of your screen.
+#. Deselect :guilabel:`Only devices advertising SMP UUID`.
+
+You should see the device node appear in the list of devices.
+
+Enabling and configuring the extended advertising API
+*****************************************************
+
+Enabling and configuring the extended advertising API for your Bluetooth mesh sample, in a way that makes the code from the :ref:`zephyr:smp_svr_sample` sample work with the mesh, will make the device node discoverable by the `nRF Connect Device Manager`_ mobile app with the :guilabel:`Only devices advertising SMP UUID` filter enabled.
```

Comment:
> Enabling and configuring the extended advertising API for your Bluetooth mesh sample, in a way that makes the code from the :ref:`zephyr:smp_svr_sample` sample work

I might have confused you with my initial comment. smp svr sample doesn't enable the extended advertising API, it just advertises SMP UUID. But to make smp svr sample code work with mesh, we need to enable the extended advertising API. My brief suggestion:

``` suggestion
Advertising SMP UUID
====================

1. Enable and configure the extended advertising API for your Bluetooth mesh sample:

...

2.  Use the code from the :ref:`zephyr:smp_svr_sample` sample to advertise SMP UUID

...
```

But few things have changed since my initial comment:
1. Extended advertising API is default now, so not sure if it needs to be mentioned in the guide. Probably as a note?
2. Now we have added SMP UUID advertisements implemented in light sample, so I guess we could point to this sample instead.